src/site/teasers/news_story_page_feature.drupal.liquid

```diff
@@ -3,7 +3,7 @@
 {% endif %}
 {% assign isStoriesPage = entityUrl.path | isPage: "stories" %}
 
-<div data-template="teasers/news_story_page_feature" id="featured-content" class="usa-grid usa-grid-full vads-u-margin-bottom--3 medium-screen:vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row vads-u-border-left--7px vads-u-border-color--primary-alt-lightest">
+<div data-template="teasers/news_story_page_feature" id="featured-content-{{ node.entityId }}" class="usa-grid usa-grid-full vads-u-margin-bottom--3 medium-screen:vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column medium-screen:vads-u-flex-direction--row vads-u-border-left--7px vads-u-border-color--primary-alt-lightest">
```

Comment:
Is it possible for `node` to be undefined?

Reply:
I don't think for `featured-content` but I can check